config/development.json

```diff
@@ -70,8 +70,6 @@
 		"guided-tours/design-showcase-welcome": true,
 		"guided-tours/site-title": true,
 		"guided-tours/theme-sheet-welcome": true,
-		"gutenberg": true,
-		"gutenberg/opt-in": true,
```

Comment:
We might still want to keep this flag around and not tie the functionality behind it to `isGutenbergEnabled` check. At one point in the future we'll be removing the opt-in and switching users to opt-out mode. With that in mind, it would be easier to make that transition by just turning this flag off (and cleaning up the code afterwards).